src/diagnostics/LogPointsManager.ts

```diff
@@ -68,6 +68,12 @@ class DebugSessionManager {
         logpointsCollection.updateTextEditorDecroration();
     }
 
+    public removeLogpointGlyphsFromDocument(documentUri: vscode.Uri): void {
+        const logpointsCollection = this.getLogpointCollectionForDocument(documentUri);
```

Comment:
This could be undefined if the input was bad?

Reply:
The `getLogpointCollectionForDocument()` is responsible for creating a collection if it is not there. So `logpointsCollection ` can be guaranteed to non-undefined.